modules/apps/frontend-js/frontend-js-translation-web/types/src/main/resources/META-INF/resources/index.d.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {IProps} from './TranslationSelector';
+declare global {
```

Comment:
Did you generated the `.d.ts` file and add the declaration of Language.get there?

If yes, I think We can declare Language.get in the component file(`.tsx`) which is using and just generate `.d.ts`.

Reply:
> Did you generated the .d.ts file and add the declaration of Language.get there?

As far as I can tell generating the `d.ts` file is done automagically, and yes; `Language.get` is  declared here 

> If yes, I think We can declare Language.get in the component file(.tsx) which is using and just generate .d.ts.

All components in this (OSGi) module that use strings will end up using `Liferay.Language.get` because we need i18n, and we're actually having a discussion about those globals [here](https://liferay.slack.com/archives/C3JBR21HA/p1624358954210300) on Slack, so let me know